deployment.rb

```diff
@@ -218,7 +218,7 @@ def set_slogger_for_test(slogger)
   end
 
   def slog(params)
-    return unless slog_token
+    return unless slog_token && Gatekeeper.allows('activityMonitor', default: true)
```

Comment:
I could instead say Gatekeeper.allows('slogging') if that seems better.  If no slogging happens, then no activity monitor logging will happen.